logdb/logdb.go

```diff
@@ -474,14 +487,20 @@ func (w *Writer) Write(b *block.Block, receipts tx.Receipts) error {
 
 		for clauseIndex, output := range r.Outputs {
 			for _, ev := range output.Events {
+				t0Val := topicValue(ev.Topics, 0)
```

Comment:
Looks the same as changing before.

Reply:
NVM, I got it.

Reply:
Apologies, this was me debugging I think, I'll revert